gem/templates/springster/patterns/basics/images/sp_variations/article-image.html

```diff
@@ -1,6 +1,7 @@
 {% load wagtailcore_tags wagtailimages_tags core_tags %}
 {% if self.get_effective_image %}
   <a href="{% pageurl self %}" class="article__anchor article__anchor--{{type}}">
-    {% image self.get_effective_image fill-360x160 class="article__image article__image--{{type}}"%}
+    {% image self.get_effective_image fill-360x160 as article_image %}
+    <img alt="{{ self.title }}" src="{{ article_image.url }}" class="article__image article__image--{{type}}" />
```

Comment:
@projectmushroom I don't think this pattern is necessary. See [here](http://docs.wagtail.io/en/v1.12.2/topics/images.html#more-control-over-the-img-tag) for how to do this in one line

Reply:
Wait, sorry nvm. I see we want to add additional args to the class using template variables.

Reply:
I changed it from the inline version because of the "type" value in the class name, class="article__image article__image--{{type}}

Reply:
😎